doc/haikuporter_bash_completion

```diff
@@ -41,4 +41,4 @@ function _haikuporter()
 
 	return 0
 }
-complete -F _haikuporter -o filenames haikuporter
```

Comment:
Similar thing as in the previous comment - need to figure out if this change is ok or not.

Reply:
Nów this should be fixed, but please check if I'm fixed it correctly.